### ISSUE TITLE
H2 500 Klipper config

### ISSUE THREAD

**Issue by fabio-s-franco** [bigtreetech/H2-Series#7]
One of the most frustrating experiences I have ever had was to setup the H2 500C in klipper. There is absolutely no documentation around it, not even for the manta board.

Although I mostly succeeded I still can't find any reference value for the PT100 thermocouple so that it can be used in hardware mode, instead of software mode.

Least I would expect are the values for [adc_temperature] as I don't have any reliable means to calibrate these.

**Comment by NexGen-3D-Printing** [comment]
Rant Mode On......

Or basic recommended settings like:

- Recommended Retract Speed
- Recommended Unretract Speed
- Recommended Extruder Acceleration
- Example Klipper Configuration For Each Version

This is typical of these Chinese based companies, releasing what looks like pretty good hardware and giving the community zero guidance, like they have never used/tested their own product, here in the western world, we would be publicly shamed, drawn and quarter then stoned while hung from lamp post if we were to release a product like this and gave no information about it.

I have one of the very first units they released, then after that I have about four more, each one is slightly different, even gear teeth changed at one point, so at least they are improving the hardware, but since that first unit, there is still no documentation .

One more thing they still have not figured out...backlash adjustment, it needs backlash adjustment FFS, click, click, click, click, I'm used to it now, but really?

........Rant Mode Off

**Comment by masternappa916** [comment]
Yeah I still haven't managed to use the PT100 thermocouple on my ender 3 max neo  I have octopus pro 1.1 and hermit crab 2 can ver  with the H2 500c no matter which way I tried it was always giving me error after error finally I ended up using  EPCOS 100K B57560G104F  on it just to get klipper to start up   have yet to print anything successfully it keeps clogging B4 the first layer is finished
